releasenotes/notes/vf2post-no-opt-if-layout-opt-set-b9998e4ba8c5280f.yaml

```diff
@@ -0,0 +1,23 @@
+---
+fixes:
+  - |
+    Fixed an issue with the :func:`.transpile` function and the generated pass manager
+    objects returned by the :func:`.generate_preset_pass_manager` function when
+    ``optimization_level`` argument is set as ``optimization_level=3`` and either an
+    ``initial_layout`` or ``layout_method`` argument are set. The intent of the
+    ``initial_layout`` argument or the ``layout_method`` arguments are for the transpiler
+    to use those arguments for the layout and not use any other layout passes to try
+    and use a different layout. However, for ``optimization_level=3`` the
+    :class:`.VF2PostLayout` transpiler pass was being run at the end of the ``optimization``
+    stage regardless of those arguments which could potentially lead to the transpiler
+    selecting a different layout than was requested. This has been fixed so the pass is not
+    run when those arguments are set
+    Fixes `#14867 <https://github.com/Qiskit/qiskit/issues/14867>`__
+  - |
+    Fixed an issue with the :func:`.transpile` function and the generated pass manager
+    objects returned by the :func:`.generate_preset_pass_manager` function when
+    ``optimization_level`` argument is set as ``optimization_level=3``. Previously, the
+    :class:`.VF2PostLayout` transpiler pass was run at the end of the ``optimization`` stage
+    by default but the results from that were never applied to the circuit. If the pass managed
+    to find a better layout it would never be used. This has been corrected so that an improved
+    layout found in the optimization stage is applied to the cirucit.
```

Comment:
Should the release note now frame things as being a new feature for O3?

The bugfix bit of it (don't explode to factorial runtime when there are free qubits) is still worthy of a release note, but we didn't hit that path in `transpile`, relative to the parent release of this note.